discovery/openstack/instance.go

```diff
@@ -45,21 +45,22 @@ const (
 
 // InstanceDiscovery discovers OpenStack instances.
 type InstanceDiscovery struct {
-	authOpts *gophercloud.AuthOptions
-	region   string
-	interval time.Duration
-	logger   log.Logger
-	port     int
+	authOpts     *gophercloud.AuthOptions
+	region       string
+	interval     time.Duration
+	logger       log.Logger
+	port         int
+	endpointType string
```

Comment:
The underlying setting is called Availability, why is this different?

Reply:
I think endpoint_type is the more usual term. It is used for example in the nova command line client. For openstack users it is probably more familiar than availability.